main.cpp

```diff
@@ -330,10 +331,12 @@ bool PersonTwo::Eye::close(bool awake)
     return awake == false ? true : false;
 }
 
-// void myPerson::Eye::see(std::string vision, std::string typeOfColorBlindness)
-// {
+void PersonTwo::Eye::see()
+{
         // I couldn't quite figure this one out.  Maybe I can get some pointers for my code review
-// }
+        // I want the function to have the person see based on the relevant attributes of the eye struct
```

Comment:
ok.
make it happen in the code
- [ ] revise

Reply:
thanks for the push, I got it sorted.  Originally I was trying to feed in the data members like `void PersonTwo::Eye::see(std::string vision = "20/50")` The compiler didn't like this, plus I was really confusing myself!